packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -446,36 +453,59 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
     }
     assert(selectedNode == items[selectedNodeIndex]);
 
-    if (event.logicalKey == LogicalKeyboardKey.arrowDown ||
-        event.logicalKey == LogicalKeyboardKey.arrowUp) {
-      _moveSelection(event.logicalKey, scrollController, constraints.maxHeight);
+    if (event.logicalKey == LogicalKeyboardKey.arrowDown) {
```

Comment:
let's use a switch statement instead 
`switch (event.logicalKey) { ... }`

Reply:
I actually tried to do this, and got an error: `The switch case expression type 'LogicalKeyboardKey' can't override the == operator.`

LMK if there is a way to get around this.

Reply:
strange. not a blocker feel free to land without this change

Reply:
I moved things around to chain the if/else blocks more obviously.